source/jormungandr/tests/journey_common_tests.py

```diff
@@ -376,11 +376,22 @@ def test_sp_to_sp(self):
 
         # with street network desactivated
         response = self.query_region(query + "&max_duration_to_pt=0")
-        assert('journeys' not in response)
+        journeys = response.get('journeys')
+        current_region = list(self.krakens_pool)[0]
+        current_scenario = self.data_sets.get(current_region, {}).get('scenario')
+        if current_scenario in ('default', 'new_default'):
+            assert not journeys
+        elif current_scenario == 'experimental':
+            assert len(journeys) == 1
+            assert 'non_pt_walking' in journeys[0]['tags']
 
         # with street network activated
         response = self.query_region(query + "&max_duration_to_pt=1")
-        eq_(len(response['journeys']), 1)
+        if current_scenario in ('default', 'new_default'):
+            eq_(len(response['journeys']), 1)
+        elif current_scenario == 'experimental':
+            eq_(len(response['journeys']), 2)
+
```

Comment:
Sorry, that's not the problem. But we shouldn't have this journey if we want to be isofunctional. Maybe that should be done in another ticket.